package.json

```diff
@@ -13,12 +13,21 @@
   "engines": {
     "node": ">=4"
   },
+  "scripts": {
+    "commitmsg": "conventional-changelog-lint -e",
+    "precommit": "npm test",
```

Comment:
`precommit` and `prepush` will probably slow those operations down too much. circleci achieves nearly the same goal as prepush. let's just stick with `commitmsg`.